.bazelci/android-studio.yml

```diff
@@ -10,3 +24,56 @@ platforms:
       - --test_output=errors
     test_targets:
       - //:aswb_tests
+  Android-Studio-internal-under-dev:
```

Comment:
We might have misunderstood each other. I meant leaving this name as canary (as this version is officially Android Studio Canary) and to rename the tasks for IntelliJ UE+CE instead (e.g. from IntelliJ-UE-internal-canary to IntelliJ-UE-internal-under-dev). Can you please adjust the names?

Reply:
Sorry about that :) I updated it now.